src/plugins/inputs/index.js

```diff
@@ -0,0 +1,53 @@
+const clientSideCommands = ["/collapse", "/expand"];
+
+const passThroughCommands = [
+	"/as",
+	"/bs",
+	"/cs",
+	"/ho",
+	"/hs",
+	"/join",
+	"/ms",
+	"/ns",
+	"/os",
+	"/rs",
+];
+
+const userInputs = [
+	"action",
+	"away",
+	"ban",
+	"connect",
+	"ctcp",
+	"disconnect",
+	"ignore",
+	"invite",
+	"kick",
+	"kill",
+	"list",
+	"mode",
+	"msg",
+	"nick",
+	"notice",
+	"part",
+	"quit",
+	"raw",
+	"rejoin",
+	"topic",
+	"whois",
+].reduce(function(plugins, name) {
+	const plugin = require(`./${name}`);
+	plugin.commands.forEach((command) => (plugins[command] = plugin));
+	return plugins;
+}, {});
+
+const getCommands = () =>
+	Object.keys(userInputs)
+		.map((command) => `/${command}`)
+		.concat(clientSideCommands)
+		.concat(passThroughCommands);
```

Comment:
This probably should be sorted.

Reply:
What you referring to?

Reply:
The final constructed array should be sorted if we want to match the previously hardcoded array command we had.

Reply:
Ahhh sorted, gotcha. Yup, good catch.